scripts/add-cluster.sh

```diff
@@ -24,181 +24,16 @@ login_to_cluster() {
   fi
 }
 
-create_service_account() {
-# we need to delete the bindings since we cannot change the roleRef of the existing bindings
-if [[ -n `oc get rolebinding ${SA_NAME} 2>/dev/null` ]]; then
-    oc delete rolebinding ${SA_NAME} -n ${OPERATOR_NS} ${OC_ADDITIONAL_PARAMS}
-fi
```

Comment:
Good point @rajivnathan !

TBH I wasn't aware of this limitation - and this might complicate things a bit since the resource controller is not aware of which resources is creating, it should be generic.

Maybe @MatousJobanek might know better if this is still required ?

Reply:
The deletion part was there just to make sure that the RoleBinding can be updated also in the cases when we would change the name of the Role that is referenced there - you know, it's not possible to update `roleRef` for an already existing RoleBinding. 
I guess that this is from the time when we were doing many changes in the content of the script/resources. 

That being said, I think that it's fine to keep it as it is in the Toolchaincluster resources controller :+1: we can change it later in the case when we would need to update the `roleRef`.